go-controller/hybrid-overlay/pkg/controller/node_windows.go

```diff
@@ -52,7 +175,226 @@ func (n *NodeController) Sync(nodes []*kapi.Node) {
 
 // InitSelf initializes the node it is currently running on.
 // On Windows, this means:
-//  1. Setting up this node and its VXLAN extension for talking to other nodes
+//  1. Setting up this node and its VxLAN extension for talking to other nodes
 //  2. Setting back annotations about its VTEP and gateway MAC address to its own node object
+//  3. Initializing every VXLAN tunnels toward other
 func (n *NodeController) InitSelf() {
+	// Retrieve the host prefix from the annotations
+	hostsubnet, ok := n.thisNode.Annotations[types.HybridOverlayNodeSubnet]
+
+	if !ok {
```

Comment:
shouldn't have blank line between a command and its error check. (likewise immediately below)